api/HTMLElement.json

```
@@ -288,6 +288,44 @@
           }
         }
       },
+      "autocorrect": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#dom-autocorrect",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "14.1",
+              "partial_implementation": true,
+              "notes": [
+                "Takes values of <code>true</code>/<code>false</code> (instead of <code>on</code>/<code>off</code>.",
+                "Allowed on <code>&lt;input type='password'&gt;</code> elements."
+              ]
```

My thoughts after testing it in latest Safari:

1. Safari accepts "true" and "false" as aliases of "on" and off", but I wouldn't consider this alone a partial implementation.
2. However, Safari returns "true" and "false" instead of "on" and "off" and I think that's the main issue here, and warrants the partial_implementation.
3. Allowing the attribute on password fields is not spec-compliant, but doesn't warrant a note, if you ask me.

```suggestion
              "notes": "Returns <code>true</code>/<code>false</code> (instead of <code>on</code>/<code>off</code>.",
```

---

@caugner Thanks. Modifying your version slightly to capture that it both accepts and returns the wrong values. We don't need to state explicitly though.

```suggestion
              "notes": "Values are <code>true</code>/<code>false</code> (instead of <code>on</code>/<code>off</code>."
```

I am OK with the note about the password being removed. It would be significant to anyone who is relying on autocorrect being enabled on passwords but that shouldn't be anyone. Users will just start having their passwords unchanged, which is a win.